.github/workflows/cd.yml

```diff
@@ -138,40 +60,97 @@ jobs:
       - name: Install Rust toolchain
         uses: dtolnay/rust-toolchain@stable
         with:
-          toolchain: stable-msvc
-          targets: ${{ env.TARGET }}
+          toolchain: stable
+      - name: Add iOS targets
+        run: rustup target add ${{ matrix.TARGET }}
       - name: Setup caching
         uses: Swatinem/rust-cache@v2
         with:
-          key: build-${{ env.TARGET }}-${{ env.CACHE_SUFFIX }}
+          key: build-${{ matrix.TARGET }}-${{ matrix.CACHE_SUFFIX }}
       # Build
-      - name: Install llvm
-        run: |
-          set -x
-          choco install -y llvm
-          echo "LIBCLANG_PATH=C:\Program Files\LLVM\lib" >> $GITHUB_ENV
-        shell: bash
       - name: Build
         run: make package
-        shell: bash
       # Upload (same for each build)
       - name: Upload
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           path: dist
-          name: dist
+          name: ${{ env.ARCHIVE_NAME }}
 
   # -----
-  windows-i686:
-    # Config
-    name: release - windows-i686
-    runs-on: windows-latest
+  android:
+    name: release - ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - name: android-i686
+            target: i686-linux-android
+            setup_env: |
+              set -x
+              ANDROID_NDK_HOME=$ANDROID_HOME/ndk/25.2.9519653
```

Comment:
Do you think we should use 27.0.12077973 as was done in #411?

Reply:
@almarklein No regressions were observed after updating.

![image](https://github.com/user-attachments/assets/eb2097ad-7431-44a2-8f20-6f57197780d9)